src/routes/responses/responses.controller.js

```diff
@@ -0,0 +1,53 @@
+const _get = require('lodash/get');
+const db = require("./../../db/index");
+
+// Organize Response Object from Slack API
+const postResponses = function (req, res) {
+  const body = JSON.parse(req.body.payload);
+  const response = {
+    responseSelected: _get(body, 'actions[0].text.text', ''),
+    id: _get(body, 'actions[0].block_id', '')
+  };
+
+  res.set('Content-Type', 'application/json');
+  res.send(`Received new Poll: ${response}`);
+
+  updateDatabase(response);
+}
+
+const getData = response => {
+  return (
+    db
+      .collection("SLACKBOT_TEST")
+      .doc(response.id)
+      .get()
+      .then(doc => {
+        if (!doc.exists) {
+          console.log('No such document!');
```

Comment:
there should be a `throw` here (in addition to the `console.log`) so you can reliably use `.then()` and `.catch()` on this promise